src/components/Widget/IntegrationsWidget.tsx

```diff
@@ -0,0 +1,477 @@
+import {
+  Badge,
+  Card,
+  CardBody,
+  CardFooter,
+  ExpandableSection,
+  Gallery,
+  List,
+  ListItem,
+  ListVariant,
+  Spinner,
+  Tile,
+} from '@patternfly/react-core';
+import React, { FunctionComponent, useEffect, useState } from 'react';
+import { Provider } from 'react-redux';
+import IntegrationsDropdown from '../IntegrationsDropdown';
+import { CLOUD_VENDOR, COMMUNICATIONS, REDHAT_VENDOR, REPORTING, WEBHOOKS } from '../../utilities/constants';
+import { getProdStore } from '../../utilities/store';
+import ImageWithPlaceholder from '../TilesShared/ImageWithPlaceholder';
+import { AsyncComponent } from '@redhat-cloud-services/frontend-components';
+import AddSourceWizard from '../addSourceWizard';
+
+import './IntegrationsWidget.scss';
+
+interface Integration {
+  type: string;
+  sub_type?: string;
+}
+
+const integrationsData = [
```

Comment:
Can we move this to a const/utils file? So we are focused in this file only on the logic itself.